src/current/_includes/releases/v23.2/v23.2.5.md

```diff
@@ -0,0 +1,68 @@
+## v23.2.5
+
+Release Date: May 7, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-5-sql-language-changes">SQL language changes</h3>
+
+- The new [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.stats.virtual_computed_columns.enabled` enables collection of table statistics on virtual computed columns. [#120923][#120923]
+- The new [session variable]({% link {{ page.version.version}}/session-variables.md %}) `optimizer_use_virtual_computed_column_stats` configures the [optimizer]({% {{ page.version.version }}/make-queries-fast.md %}) to consider table statistics on virtual computed columns. [#121179][#121179]
+- The new `FORCE_INVERTED_INDEX` [hint]({% link {{ page.version.version }}/indexes.md %}#selection) configures the [optimizer]({% {{ page.version.version }}/make-queries-fast.md %})  to prefer a query plan scan over any inverted index of the hinted table. If no such query plan can be generated, an error is logged. [#122300][#122300]
+- The [optimizer]({% {{ page.version.version }}/make-queries-fast.md %}) can now plan constrained scans over partial indexes in more cases, particularly on partial indexes with predicates referencing virtual computed columns. [#123408][#123408]
+
+<h3 id="v23-2-5-operational-changes">Operational changes</h3>
+
+- A minimum Raft scheduler concurrency is now enforced per [store]({% link {{ page.version.version }}/cockroach-start.md %}#storage) so that a node with many stores does not spread workers too thinly. This avoids high scheduler latency across replicas on a store when load is imbalanced. [#120798][#120798]
+- A [changefeed]({% link {{ page.version.version }}/change-data-capture-overview.md %}) optimization to reduce duplicates during aggregator restarts has been disabled due to poor performance. [#123596][#123596]
+
+<h3 id="v23-2-5-db-console-changes">DB Console changes</h3>
+
+- The **Commit Latency** chart in the [Changefeed Dashboard]({% link {{ page.version.version }}/ui-cdc-dashboard.md %}) now aggregates by max instead of by sum for multi-node changefeeds. This more accurately reflects the amount of time for events to be acknowledged by the downstream sink. [#121235][#121235]
+
+<h3 id="v23-2-5-bug-fixes">Bug fixes</h3>
+
+- Fixed a slow memory leak when opening many new connections. This bug was introduced in 22.2.9 and 23.1.0. [#121055][#121055]
```

Comment:
'connections' could be a link to https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-your-cluster.html

version numbers should have a 'v' at the front, e.g. v22.2.9 and v23.1.0

Reply:
I'll link to https://www.cockroachlabs.com/docs/v23.2/connect-to-the-database.html instead of the CC page.
```suggestion
- Fixed a slow memory leak when opening many new [connections]({% link {{ page.version.version }}/connect-to-the-database.md %}. This bug was introduced in 22.2.9 and 23.1.0. [#121055][#121055]
```